get_phantom_mask_from_snr_map.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import scipy.ndimage as ndi
+
+
+def plot_3d(image):
+    # Placeholder function for 3D plotting
+    import matplotlib.pyplot as plt
+    from mpl_toolkits.mplot3d import Axes3D
+
+    fig = plt.figure()
+    ax = fig.add_subplot(111, projection='3d')
+    x, y, z = np.where(image > 0)
+    ax.scatter(x, y, z, c='red', marker='o', alpha=0.5)
+    plt.show()
+
+
+def get_phantom_mask_from_snr(SNR, thres=None):
+    if thres is None:
+        option = int(input(
+            "Determine threshold (option 1) or number of voxels in Mask (option 2) ?\n"))
+        if option == 1:
+            # plot_3d(SNR)
+            thres = float(
+                input("Enter SNR threshold to determine mask (above 1):\n"))
+            print("WARNING: generating mask with the specified SNR-threshold")
+            while thres != 1:
+                Mask = np.zeros_like(SNR, dtype=bool)
```

Comment:
Please rename to `mask`, in python you dont use uppercase for variables, its for Class names